src/frontend/src/pages/ProjectDetailPage/ProjectForm/ProjectAutoCRContainer.tsx

```diff
@@ -0,0 +1,131 @@
+import { ProjectProposedChangesCreateArgs, WbsElementStatus } from 'shared';
+import * as yup from 'yup';
+import { ProjectCreateChangeRequestFormInput } from './ProjectEditContainer';
+
+const CreateProjectCRContainer: React.FC = () => {
+  const toast = useToast();
+  const history = useHistory();
+
+  const [projectManagerId, setProjectManagerId] = useState<string | undefined>();
+  const [projectLeadId, setProjectLeadId] = useState<string | undefined>();
+
+  const { mutateAsync, isLoading } = useCreateStandardChangeRequest();
+  const {
+    data: allLinkTypes,
+    isLoading: allLinkTypesIsLoading,
+    isError: allLinkTypesIsError,
+    error: allLinkTypesError
+  } = useAllLinkTypes();
+
+  if (isLoading) return <LoadingIndicator />;
+  if (!allLinkTypes || allLinkTypesIsLoading) return <LoadingIndicator />;
+  if (allLinkTypesIsError) return <ErrorPage message={allLinkTypesError.message} />;
+
+  const requiredLinkTypeNames = getRequiredLinkTypeNames(allLinkTypes);
+
+  const defaultValues = {
+    name: '',
+    budget: 0,
+    summary: '',
+    teamIds: [],
+    carNumber: 0,
+    links: [],
+    crId: '',
+    goals: [],
+    features: [],
+    constraints: [],
+    rules: [],
+    projectLeadId,
+    projectManagerId
+  };
+
+  const schema = yup.object().shape({
+    name: yup.string().required('Name is required!'),
+    carNumber: yup.number().min(0).required('A car number is required!'),
+    teamId: yup.string().required('A Team Id is required'),
+    budget: yup.number().optional(),
+    summary: yup.string().required('Summary is required!'),
+    projectLeadId: yup.number().optional(),
+    projectManagerId: yup.number().optional(),
+    links: yup.array().of(
+      yup.object().shape({
+        linkTypeName: yup.string(),
+        url: yup.string().url('Invalid URL')
+      })
+    )
+  });
+
+  const onSubmitChangeRequest = async (data: ProjectCreateChangeRequestFormInput) => {
+    console.log('in onSubmitChangeRequest');
+    const { name, budget, summary, links, teamIds, carNumber, goals, features, constraints, type, what, why } = data;
+
+    const rules = data.rules.map((rule) => rule.detail);
+
+    try {
+      const projectPayload: ProjectProposedChangesCreateArgs = {
+        name,
+        summary,
+        status: WbsElementStatus.Active,
+        teamIds: teamIds.map((number) => '' + number),
+        budget,
+        rules,
+        goals: goals.map((g) => g.detail),
+        features: features.map((f) => f.detail),
+        otherConstraints: constraints.map((c) => c.detail),
+        links,
+        projectLeadId: projectLeadId ? parseInt(projectLeadId) : undefined,
+        projectManagerId: projectManagerId ? parseInt(projectManagerId) : undefined,
+        carNumber: carNumber
+      };
+      const changeRequestPayload: CreateStandardChangeRequestPayload = {
+        wbsNum: {
+          carNumber: carNumber,
```

Comment:
I like this, can you put in a todo here to change this to the car model in the database when we get around to adding that